### PR TITLE
Add keras mnist demo

### PR DESCRIPTION
**Pull request by daming-lu**

### CODE REVIEW

File: demo/keras/TUTORIAL_CN.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        image0.start_sampling()
+        image0.add_sample([9, 32], weight_array0)
+        image0.finish_sampling()
+
+        image1.start_sampling()
+        image1.add_sample([288, 64], weight_array1)
+        image1.finish_sampling()
+
+        train_step += 1
+        self.losses.append(logs.get('loss'))
```
Comment (cs2be) [inline demo/keras/TUTORIAL_CN.md:69]: whats this used for?

File: demo/keras/TUTORIAL_CN.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+训练结束后，各个组件的可视化结果如下：
+
+关于误差的数值图的如下：
+
+<p align=center>
+<img width="70%" src="https://github.com/daming-lu/large_files/blob/master/keras_demo_figs/keras_scalar.png?raw=true" />
+</p>
+
+训练过后的第一，第二层卷积权重图的如下：
```
Comment (Superjomn) [inline demo/keras/TUTORIAL_CN.md]: you can use an image component to visualize training image.   Both the two images lacks information.

File: demo/keras/TUTORIAL_CN.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+# create VisualDL logger
+logdir = "/workspace"
+logger = LogWriter(logdir, sync_cycle=100)
+
+# mark the components with 'train' label.
+with logger.mode("train"):
+    # create a scalar component called 'scalars/'
+    scalar_keras_train_loss = logger.scalar(
+        "scalars/scalar_keras_train_loss")
+    image0 = logger.image("images/image0", 1)
```
Comment (cs2be) [inline demo/keras/TUTORIAL_CN.md]: why do we need to prefix with "images/"?  Isn't this graph only going to show up in images section?

File: demo/keras/TUTORIAL_CN.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    image1 = logger.image("images/image1", 1)
+    histogram0 = logger.histogram("histogram/histogram0", num_buckets=50)
```
Comment (cs2be) [inline demo/keras/TUTORIAL_CN.md]: Same here, why do we need "histogram/" prefix?

File: demo/keras/TUTORIAL_CN.md (hunk 2 above)
Comment (daming-lu) [reply demo/keras/TUTORIAL_CN.md]: Done.

File: demo/keras/TUTORIAL_CN.md (hunk 3 above)
Comment (daming-lu) [reply demo/keras/TUTORIAL_CN.md]: 'images/' is showing that we can group metrics by namespace, which will be implemented in the future.

File: demo/keras/TUTORIAL_CN.md (hunk 4 above)
Comment (daming-lu) [reply demo/keras/TUTORIAL_CN.md]: ditto

File: demo/keras/TUTORIAL_CN.md (hunk 1 above)
Comment (daming-lu) [reply demo/keras/TUTORIAL_CN.md:69]: removed as it is not very related to this demo.

**Review by Superjomn — APPROVED** [review]
LGTM

**Pull request merged**